src/http/web_socket.cr

```diff
@@ -208,6 +207,16 @@ class HTTP::WebSocket
       end
     end
   end
+
+  private def do_close(code, message)
```

Comment:
I'd suggest using `protected` visibility to allow access from child classes.

ditto below

Reply:
Private _is_ accessible from subclasses.